plugins/coverbrowser.koplugin/main.lua

```diff
@@ -159,12 +160,65 @@ function CoverBrowser:addToMainMenu(menu_items)
                         },
                     },
                 },
+                -- Misc settings
+                {
+                    text = _("Other settings"),
+                    sub_item_table = {
+                        {
+                            text = _("Show hint for books with description"),
+                            checked_func = function() return not BookInfoManager:getSetting("no_hint_description") end,
+                            callback = function()
+                                if BookInfoManager:getSetting("no_hint_description") then
+                                    BookInfoManager:saveSetting("no_hint_description", false)
+                                else
+                                    BookInfoManager:saveSetting("no_hint_description", true)
+                                end
+                                self:refreshFileManagerInstance()
+                            end,
+                        },
+                        {
+                            text = _("Show hint for opened books in History"),
+                            checked_func = function() return BookInfoManager:getSetting("history_hint_opened") end,
+                            callback = function()
+                                if BookInfoManager:getSetting("history_hint_opened") then
+                                    BookInfoManager:saveSetting("history_hint_opened", false)
+                                else
+                                    BookInfoManager:saveSetting("history_hint_opened", true)
+                                end
+                                self:refreshFileManagerInstance()
+                            end,
+                        },
+                        {
+                            text = _("Append Series metadata to Authors"),
+                            checked_func = function() return BookInfoManager:getSetting("append_series_to_authors") end,
+                            callback = function()
+                                if BookInfoManager:getSetting("append_series_to_authors") then
+                                    BookInfoManager:saveSetting("append_series_to_authors", false)
+                                else
+                                    BookInfoManager:saveSetting("append_series_to_authors", true)
+                                end
+                                self:refreshFileManagerInstance()
+                            end,
+                        },
+                        {
+                            text = _("Append Series metadata to Title"),
```

Comment:
Same here, to highlight Series and Titles, just like they appear in Bookinfo kvpage.
Or I could put them in single or double quotes ?
Or just as you said, in lowercase.
Just tell me after you read my reasoning, and I'll do as you prefer.